index.js

```diff
@@ -0,0 +1,51 @@
+const path = require("path");
+const express = require("express");
+const morgan = require("morgan");
+const cookieParser = require("cookie-parser");
+const bodyParser = require("body-parser");
+
+// ROUTES
+
+const welcome = require("./routes/welcome");
```

Comment:
To someone reading your code, the variable name `welcome` doesn't really describe what this represents. `welcome_routes` might be a better variable name.